simtools/corsika_simtel/corsika_simtel_runner.py

```diff
@@ -0,0 +1,202 @@
+import stat
+from pathlib import Path
+
+from simtools.corsika.corsika_runner import CorsikaRunner
+from simtools.simtel.simtel_runner_array import SimtelRunnerArray
+
+__all__ = ["CorsikaSimtelRunner"]
+
+
+class MissingRequiredEntryInCorsikaConfig(Exception):
+    """Exception for missing required entry in corsika config."""
+
+
+class CorsikaSimtelRunner(CorsikaRunner, SimtelRunnerArray):
+    """
+    CorsikaSimtelRunner is responsible for running CORSIKA and piping it to sim_telarray
+    using the multipipe functionality. CORSIKA is set up using corsika_autoinputs program
+    provided by the sim_telarray package. It creates the multipipe script and sim_telarray command
+    corresponding to the requested configuration.
+
+    It uses CorsikaConfig to manage the CORSIKA configuration and SimtelRunnerArray
+    for the sim_telarray configuration. User parameters must be given by the
+    common_args, corsika_args and simtel_args arguments.
```

Comment:
I dont understand why so many line breaks, since it did not reach the limit and it also did not start a new sentence. Are these really necessary?

Reply:
Yes it reached the limit so I tried breaking it where it is somewhat easier to read. In `read_the_docs` these are anyway meaningless no?